modules/cli/pages/cli-intro.adoc

```diff
@@ -39,46 +39,117 @@ You can collect diagnostics through the command-line interface by using the xref
 [#server-tools-packages]
 == Server Tools Packages
 
-For convenience, Couchbase provides a  tools package  that contains the following utilities:
+For convenience, Couchbase provides Server developer tools package and Server admin tools package.
+
+=== Server Developer Tools Packages
+
+Server developer tools package contains the following utilities:
+
+* xref:tools:cbimport.adoc[`cbimport`]
+* xref:tools:cbexport.adoc[`cbexport`]
+* xref:cli:cbq-tool.adoc[`cbq`]
+* xref:backup-restore:cbbackupmgr.adoc[`cbbackupmgr`]
+* xref:cbcli:couchbase-cli.adoc[`couchbase-cli`]
```

Comment:
couchbase-cli, cbstats, mctimings, and mcstat are not in the developer tools package.
(Nothing was added to the existing developer tools package -- just the naming was changed to add the "dev" part.)